ocaml/libs/tracing/tracing.ml

```diff
@@ -138,20 +164,22 @@ module Span = struct
             )
         in
         let status_code = Status.Error in
-        let exn_tags =
+        let exn_attributes =
           [
             ("exception.message", msg)
           ; ("exception.stacktrace", stacktrace)
           ; ("exception.type", exn_type)
+          ; ("error", "true")
           ]
         in
         match span.status.status_code with
         | Unset ->
-            {
-              span with
-              status= {status_code; description}
-            ; tags= span.tags @ exn_tags
-            }
+            let attributes =
+              List.fold_left
+                (fun attrs (k, v) -> Attributes.add k v attrs)
+                Attributes.empty exn_attributes
```

Comment:
The previous code united span.tags and exn_tags. it now unites exn_attributes with en empty map, this doesn't look like it's intended

Reply:
That was a copy paste error, now adjusted